packages/chevrotain/docs/features/gates.md

```diff
@@ -20,5 +20,23 @@ $.RULE("Value", isConst => {
 })
 ```
 
+Using the [Look Ahead](https://sap.github.io/chevrotain/documentation/6_5_0/classes/cstparser.html#la) method is often helpful with the use of Gates to determine if a path should be followed or not, for example:
```

Comment:
Feels like this should be expended to a fuller example of resolving ambiguities that are created because the parser is greedy and only linked to from this doc.

Maybe another section here: https://sap.github.io/chevrotain/docs/guide/resolving_grammar_errors.html ?

Reply:
Or maybe a section in the FAQ:
- https://sap.github.io/chevrotain/docs/FAQ.html